apps/dotcms-ui/src/app/view/components/dot-edit-layout-designer/dot-edit-layout-designer.component.ts

```diff
@@ -68,15 +68,20 @@ export class DotEditLayoutDesignerComponent implements OnInit, OnDestroy, OnChan
     @Output()
     save: EventEmitter<Event> = new EventEmitter();
 
+    @Output()
+    saveAndPublish: EventEmitter<Event> = new EventEmitter();
```

Comment:
Isn't this a `MouseEvent`?

Reply:
no, it sends the same as `save` the form value